.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,29 @@
+name: Notify Maintainer
+
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches:
+      - main
+
+jobs:
+  notify:
+    name: Send Slack Notification
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Send GitHub Action trigger data to Slack workflow
+      id: slack
+      uses: slackapi/slack-github-action@v1.21.0
+      with:
+        payload: |
+          {
+            "repoName": "${{ github.repository }}",
+            "prNumber": "${{ github.event.pull_request.number }}",
+            "requester ": "${{ github.actor }}",
+            "link": "${{ github.server_url }}/${{ github.repository }}/pull/${{ github.event.pull_request.number }}"
+          }
+      env:
+        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Should we switch this to the org secret?

Reply:
`SLACK_WEBHOOK_URL` has been created on this repo, so it shouldn't be an issue.

Reply:
I didn't like it being an org secret because of the public nature.

Maybe that's silly.